src/plugins/vis_type_drilldown/public/drilldown_options.tsx

```diff
@@ -12,23 +12,72 @@ import {
   EuiFlexItem,
   EuiFieldText,
   EuiAccordion,
+  EuiSuperSelect,
+  EuiText,
 } from '@elastic/eui';
-import { FormattedMessage } from '@osd/i18n/react';
 
 import { VisOptionsProps } from 'src/plugins/vis_default_editor/public';
-import { DrilldownVisParams } from './types';
+import { useOpenSearchDashboards } from 'src/plugins/opensearch_dashboards_react/public';
+import { DrilldownServices, DrilldownVisParams } from './types';
 
 function DrilldownOptions({ stateParams, setValue }: VisOptionsProps<DrilldownVisParams>) {
   const onMarkdownUpdate = useCallback(
     (value: DrilldownVisParams['cardName']) => setValue('cardName', value),
     [setValue]
   );
 
+  const {
+    services: { savedObjectsClient },
+  } = useOpenSearchDashboards<DrilldownServices>();
+
+  useEffect(() => {
+    const saved = savedObjectsClient.find({
```

Comment:
That's not quite the right format for the service. It should be:
```suggestion
    services: { savedObjects },
  } = useOpenSearchDashboards<DrilldownServices>();

  useEffect(() => {
    const saved = savedObjects?.client.find({
```